tests/test_pi_extension_install.py

```
@@ -86,6 +91,47 @@ def main() -> int:
 
+        extension_path.write_text(
+            "// cmux-pi-session-extension-marker v2\n"
+            "// stale managed fixture using synchronous hook dispatch\n"
+            'import { spawnSync } from "node:child_process";\n',
+            encoding="utf-8",
+        )
+        refresh_env = os.environ.copy()
+        refresh_env["PI_CODING_AGENT_DIR"] = str(config_dir)
+        refresh_env["CMUX_WORKSPACE_ID"] = FOCUSED_WORKSPACE_ID
+        refresh_env["CMUX_SURFACE_ID"] = FOCUSED_SURFACE_ID
+        subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)